### PR TITLE
Add post test plots for performance test

### PR DESCRIPTION
**Pull request by ahcorde**
This changes that @cottsay has done allow to generate plots that compared results between different DDS vendors

Some results: http://build.ros2.org/view/Eci/job/Eci__nightly-performance_ubuntu_bionic_amd64/plot/

### CODE REVIEW

File: doc/configuration_options.rst [hunk 1]
```diff
@@ -452,6 +452,28 @@ The following options are valid in version ``1`` (beside the generic options):
...
+* ``show_plots``: a dictionary of lists, where the key is the title of the
+  plot group and the list contains plot definitions comprised of:
+
+  * ``title``: the title of the plot.
+  * ``y_axis_label``: (optional) a label for the y-axis.
+  * ``master_csv_name``: the name of the CSV file in which to aggregate the
+    results on the Jenkins master.
+    It must be unique among all plot instances on the same Jenkins master.
+  * ``style``: the type of plot used to display the data.
+  * ``y_axis_exclude_zero``: if True, the plot may not include a zero value
```
Comment (dirk-thomas) [inline doc/configuration_options.rst]: Similar to other config options mention the type, e.g.:

> ``y_axis_exclude_zero``: a boolean flag if zeros should be excluded

File: doc/configuration_options.rst [hunk 2]
```diff
@@ -452,6 +452,28 @@ The following options are valid in version ``1`` (beside the generic options):
...
 
+* ``show_plots``: a dictionary of lists, where the key is the title of the
+  plot group and the list contains plot definitions comprised of:
+
+  * ``title``: the title of the plot.
+  * ``y_axis_label``: (optional) a label for the y-axis.
+  * ``master_csv_name``: the name of the CSV file in which to aggregate the
+    results on the Jenkins master.
+    It must be unique among all plot instances on the same Jenkins master.
+  * ``style``: the type of plot used to display the data.
```
Comment (dirk-thomas) [inline doc/configuration_options.rst]: Please enumerate the allowed values.

Same below for `data_type`, `selection_flag`, and `selection_value`.

Comment (cottsay) [reply doc/configuration_options.rst]: I did this for `style`, `data_type` and `selection_flag`, but `selection_value` is hard. It differs completely based on `selection_flag`, and I'd prefer not to write out each one.

If this isn't sufficient, I'm tempted to just remove the options until we actually want them to be different. All of the use cases for the performance jobs are currently setting the same value for each.

Comment (dirk-thomas) [reply doc/configuration_options.rst]: Maybe list at least some examples? As a reader I have no idea what is expected here.

Comment (cottsay) [reply doc/configuration_options.rst]: Added.

Comment (dirk-thomas) [reply doc/configuration_options.rst]: The text itself looks good. Please start each sentence on a new line.

**Pull request merged**